### PR TITLE
Add consent checkbox

### PR DESCRIPTION
**Pull request by luthfib**
Adds a consent checkbox to the patient request drawer

image

Still need: 
 - [ ] Text for the checkbox in the patient request drawer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ const onPatientSave = async (data: PatientFormData) => {
     city: string;
     state: string;
     zipCode: string;
+    consent: string;
```
Comment (WesleyKapow) [inline README.md:72]: `onPatientSave` doesn't care about `consent` and shouldn't receive it, just like `npi`.

File: src/components/core/checkbox.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { useState } from "react";
+
+export type CheckboxProps = {
+  name: string;
+  text: string;
+};
+
+export function Checkbox({ name, text }: CheckboxProps) {
```
Comment (WesleyKapow) [inline src/components/core/checkbox.tsx:8]: Hmm not sure what the right name for this is. It's really a checkbox box as it not only has a checkbox ☑️, but wraps it in a box with some text.

Maybe `ConsentField` -- `core/forms/consent-field.tsx`?

**Review by WesleyKapow — CHANGES_REQUESTED** [review]

**Review by WesleyKapow — APPROVED** [review]
LGTM

1. `ConsentField` should probably be in `core/form` directory.
2. Still have the todo around the message in that consent box.

**Comment by luthfib** [comment]
We have decided not to do this

**Pull request closed without merging**